<launch>
####################Needs an rqt_image_view for each topic created in the homework
<node pkg="image_work" name="hw7node" type="hw7node.py"/>
<node pkg="image_work" name="hw8node" type="hw8node.py"/>
<node pkg="image_processing_hw" name="image_pub_all" type="image_pub_all.py"/>
<node pkg="rqt_image_view" name="image_view_edges" type="rqt_image_view" args="/image_edges" />
<node pkg="rqt_image_view" name="image_view_white_lines" type="rqt_image_view" args="/image_lines_white" />
<node pkg="rqt_image_view" name="image_view_yellow_lines" type="rqt_image_view" args="/image_lines_yellow" />

</launch>
